charts/ceph-csi-rbd/templates/secret.yaml

```diff
@@ -4,6 +4,9 @@ kind: Secret
 metadata:
   name: {{ .Values.secret.name }}
   namespace: {{ .Release.Namespace }}
+  {{- with .Values.secret.annotations }}
+  annotations: {{- . | toYaml | nindent 4 }}
```

Comment:
can you please do similar change for cephfs secret as well?

Reply:
done